spec/unit/blockscore/candidate_spec.rb

```diff
@@ -1,46 +1,192 @@
+    describe '#refresh' do
+      subject(:candidate) { create(:candidate, name_last: 'Smith') }
+      before do
+        candidate.name_last = 'John'
+        candidate.refresh
+      end
 
-      it { expect(candidate.name_first).to eql('Chris') }
-      include_context 'api request'
+      it { expect(candidate.name_last).to eq 'Smith' }
     end
 
-    describe '#delete' do
-      before { candidate.delete }
+    describe '#inspect' do
+      subject(:candidate_inspection) { create(:candidate).inspect }
+      it { expect(candidate_inspection.class).to be String }
+    end
 
-      let(:uri)         { "/candidates/#{candidate.id}" }
-      let(:http_method) { :delete                       }
+    pending '#update' do
+      subject(:candidate) { create(:candidate, name_first: 'John') }
+      before do
+        candidate.update(name_first: 'Jane')
+      end
 
-      include_context 'api request'
+      it { is_expected.to be_persisted }
+      it { expect(candidate.name_first).to eq 'Jane' }
     end
 
-    describe '#search' do
-      let(:constraints) { { name_first: 'John' } }
-      subject(:search) { -> { candidate.search(constraints) } }
+    describe '#delete' do
+      subject(:candidate) { create(:candidate) }
+      let(:candidate_id)  { candidate.id }
+      before do
+        candidate.delete
+      end
+
+      it { is_expected.not_to be_persisted }
+      it { expect { BlockScore::Candidate.retrieve(candidate_id).force! }.to raise_error BlockScore::NotFoundError }
+    end
+
+    describe '#history' do
+      subject(:candidate) { create(:candidate, name_first: 'version_0') }
+      before do
+        candidate.name_first = 'version_1'
+        candidate.save
+
+        candidate.name_first = 'version_2'
+        candidate.save
+      end
 
-      context 'search request' do
-        let(:uri)      { '/watchlists'                                         }
-        let(:body)     { { 'candidate_id' => candidate.id }.merge(constraints) }
-        let(:expected) { { body: hash_including(body) }                        }
-        before { search.call }
+      it { expect(candidate.history).not_to be_empty }
+      it { expect(candidate.history[0].class).to be BlockScore::Candidate }
+      it { expect(candidate.history[0].name_first).to eq 'version_2' }
+      it { expect(candidate.history[1].name_first).to eq 'version_1' }
+      it { expect(candidate.history[2].name_first).to eq 'version_0' }
+    end
+
+    describe '#hits' do
+      subject(:candidate) { create(:watched_candidate) }
+      before do
+        candidate.search
 
-        include_context 'POST'
+        # ==================================================
+        # Micro Patch: Spinlock until delay job has processed.
+        #
+        # After a new recording, the empty responses should be edited out to
+        #   prevent baking in the sleeps, as they will be replayed in order.
+        VCRHelper.spinlock_until { candidate.hits.count != 0 }
```

Comment:
I like the use of the helper here.  This is much more out of the way than I expected.